.github/workflows/tests.yml

```diff
@@ -194,6 +194,9 @@ jobs:
         if: matrix.python-version == '3.13.0-alpha - 3.13.0'
         run: |
           # Install to collect dependencies into the (pip) cache.
+          # cffi will probably have no public release until a Python 3.13 beta
+          # or even RC release, see https://github.com/python-cffi/cffi/issues/23
+          pip install -U "cffi @ https://github.com/python-cffi/cffi/archive/refs/heads/main.zip"
```

Comment:
This does not help 😢

We depend on `persistent`. While building the wheel for `persistent` pip downloads the latest released version of `cffi` (as it is a setup dependency) instead of using the version we are installing here. 😡 

So we have to wait until there is a released version of `cffi` which includes Python 3.13 support.